konan_cli/main.py

```diff
@@ -29,21 +28,27 @@ def konan(ctx, version):
 
 
 @konan.command()
-@click.option('--email', prompt="Email", help="The email you registered with on Konan", required=True, type=click.STRING)
-@click.option('--password', prompt="Password", help="The password of your registered user on Konan", required=True, hide_input=True, type=click.STRING)
-def login(email, password):
+@click.option('--email', prompt="Email", help="The email you registered with on Konan", required=True,
+              type=click.STRING)
+@click.option('--password', prompt="Password", help="The password of your registered user on Konan", required=True,
+              hide_input=True, type=click.STRING)
+@click.option('--api_key', prompt="Password", help="The api-key of your registered user on Konan", required=False,
```

Comment:
For consistency, can we have it `api-key` instead of `api_key`? Check the `set` command for reference.